pkg/cmd/server/api/v1/types_test.go

```diff
@@ -118,6 +118,8 @@ imageConfig:
   format: ""
   latest: false
 kind: MasterConfig
+kubeAdmissionControlPluginsOverride:
```

Comment:
if anywhere, this should be under kubernetesMasterConfig... can you already do this with `--admission-control` under apiServerArguments?